### PR TITLE
Re-former

### PR DESCRIPTION
**Pull request by AbdelrhmanAmin**
In this project, I've created a simple form to save user data
-validations
-associations
-controllers
-Delete
-New
-Edit

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to root_path
+    else
```
Comment (SpaYco) [inline app/controllers/users_controller.rb]: * [ ] `redirect_to new_user_path` if user was saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  def edit
+    @user = User.find([params[:id]])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to root_path
```
Comment (SpaYco) [inline app/controllers/users_controller.rb]: * [ ] `redirect_to edit_user_path` if user was saved

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] There should be absolutely NO logic in views. >Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

- [ ] check the rest below 👇 

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Abdo Amin Amin** 
Excellent job following the instruction of the previous TSE :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
Just one last details before you can go ahead:

- [ ] Add a screenshot to the README file 
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**